templates/rsyslog.conf.erb

```diff
@@ -67,6 +67,13 @@ $Umask <%= scope.lookupvar('rsyslog::umask') %>
 $OmitLocalLogging on
 <% end -%>
 
+# Settings for imjournal
+<% if scope.lookupvar('rsyslog::im_journal_statfile') -%>
+$IMJournalStateFile <%=scope.lookupvar('rsyslog::im_journal_statfile') %>
```

Comment:
Cool, thanks :) I've got these changes in a test env at the moment, all looking good - it's the only way I could get a newer format JSON template to work.